Core/src/org/sleuthkit/autopsy/datamodel/FileTypesByMimeType.java

```diff
@@ -170,12 +146,59 @@ private void populateHashMap() {
     }
 
     FileTypesByMimeType(SleuthkitCase skCase) {
+        this.pcl = (PropertyChangeEvent evt) -> {
+            String eventType = evt.getPropertyName();
+            if (eventType.equals(IngestManager.IngestJobEvent.COMPLETED.toString())
+                    || eventType.equals(IngestManager.IngestJobEvent.CANCELLED.toString())) {
+
+                /**
+                 * Checking for a current case is a stop gap measure until a
+                 * different way of handling the closing of cases is worked out.
+                 * Currently, remote events may be received for a case that is
+                 * already closed.
+                 */
+                try {
+                    Case.getCurrentCase();
+                    shouldShowCounts();
+
+                    populateHashMap();
+                } catch (IllegalStateException notUsed) {
+                    /**
+                     * Case is closed, do nothing.
+                     */
+                }
+            } else if (eventType.equals(Case.Events.CURRENT_CASE.toString())) {
+                if (evt.getNewValue() == null) {
+                    removeListeners();
+                }
+            }
+        };
         IngestManager.getInstance().addIngestJobEventListener(pcl);
         Case.addPropertyChangeListener(pcl);
         this.skCase = skCase;
         populateHashMap();
     }
 
+    /**
+     * Should the nodes show counts?
+     *
+     *
+     * @return True, unless the DB has more than 200k rows.
+     */
+    private boolean shouldShowCounts() {
+        if (showCounts) {
+            try {
+                if (skCase.countFilesWhere("1") > 200000) {
```

Comment:
This results in an exception in PostgreSQL...argument of where must be type boolean, not integer type. I believe that changing this to "1 = 1" works for both SQLite and PostgreSQL.